tests/components/radial-tests.js

```diff
@@ -0,0 +1,73 @@
+import test from 'tape';
+import React from 'react';
+import {mount} from 'enzyme';
+import RadialChart from 'radial-chart';
+import SimpleRadialChart from '../../showcase/radial-chart/simple-radial-chart';
+import DonutChart from '../../showcase/radial-chart/donut-chart';
+import CustomRadiusRadialChart from '../../showcase/radial-chart/custom-radius-radial-chart';
+
+import {testRenderWithProps} from '../test-utils';
+
+/* eslint-disable max-len */
```

Comment:
Maybe create a new `.eslintrc` in the `tests` folder with this rule, to be able to remove this, and also prevent all other unnecessary newlines that makes the code not really readable. If the line is really too long, we can split like this

```
t.equal(
  yolo,
  'je suis yolo',
  'the yolo should be yolo'
)